src/components/interface/Transactions.vue

```diff
@@ -130,18 +128,30 @@
 <script lang="ts">
 import { computed, defineComponent, onMounted, reactive } from "vue";
 import { BigNumber } from "bignumber.js";
+import { AccountId, Timestamp } from "@hashgraph/sdk";
 
-import { Transfer } from "../../pages/interface/Send.vue";
 import { useStore } from "../../store";
-import { TransactionRecord } from "../../services/impl/hedera/client/get-account-records";
+import { CryptoTransfer } from "../../domain/CryptoTransfer";
 
-// import Transaction from "./Transaction.vue";
 import Hint from "./Hint.vue";
 import KabutoLink from "./KabutoLink.vue";
+import Transaction from "./Transaction.vue";
+
+
+interface Transfer {
+  to?: AccountId;
+  asset: string; // "HBAR" or token ID (string)
+  amount?: BigNumber;
+  usd?: string;
+}
```

Comment:
Just in case

```suggestion
//TODO: Replace with Transfer in domain from Send PR after merge, Transfer will be in domain
interface Transfer {
  to?: AccountId;
  asset: string; // "HBAR" or token ID (string)
  amount?: BigNumber;
  usd?: string;
}
```

Reply:
Added comment